packages/@ourworldindata/utils/src/MarkdownTextWrap/parser.ts

```diff
@@ -326,7 +325,11 @@ const detailOnDemandContentParser: (
         r.nonBracketWord
     )
 
-export const detailOnDemandRegex = /\(hover::(\w+)::(\w+)\)/
+export const detailOnDemandRegex = /\(#dod:(\w+)\)/g
```

Comment:
This is a different regex than in `htmlToEnriched` 😱

Reply:
Oops! Extracted them to a single value in `utils/src/GdocsUtils.ts`